### ISSUE TITLE
Typo mistake

### ISSUE THREAD

**Issue by hasam6400** [Juhi-Purswani/Multimodal_Meta_Learning_with_Siamese_Network#5]
In file features.py, you wrote
if(img_y==-1 or aud_y == -1):
but it should be 
if(img_y[i]==-1 or aud_y[i] == -1):

correct me if i am wrong.

**Comment by Juhi-Purswani** [comment]
Yep, I suppose that would be a typo. If you want, go ahead and create a PR, else I would update it. Thanks for bringing it to my notice :).

**Comment by udhayavaran** [comment]
Hi, I will create a PR for this right now as discussed!
